healthcheck/diskcheck_js.go

```diff
@@ -0,0 +1,13 @@
+package healthcheck
```

Comment:
Similarly I think we'd need to add `// +build js` here?

Reply:
See comment above.